commit 7d184e288a688f4878e73ad80cd6d87ad44bf0f0
Author: Konstantin Kolinko <kkolinko@apache.org>
Date:   Wed Dec 3 02:01:47 2014 +0000

    Fix trailing whitespace to make Checkstyle happy. Mark fields as volatile/final. Followup to r1642962, r1642974.
    
    git-svn-id: https://svn.apache.org/repos/asf/tomcat/trunk@1643041 13f79535-47bb-0310-9956-ffa450edef68

diff --git a/test/org/apache/tomcat/websocket/pojo/TestEncodingDecoding.java b/test/org/apache/tomcat/websocket/pojo/TestEncodingDecoding.java
index 27978eb..0a760ce 100644
--- a/test/org/apache/tomcat/websocket/pojo/TestEncodingDecoding.java
+++ b/test/org/apache/tomcat/websocket/pojo/TestEncodingDecoding.java
@@ -49,6 +49,7 @@ import javax.websocket.server.ServerEndpointConfig;
 
 import org.junit.Assert;
 import org.junit.Test;
+
 import org.apache.catalina.Context;
 import org.apache.catalina.servlets.DefaultServlet;
 import org.apache.catalina.startup.Tomcat;
@@ -409,8 +410,8 @@ public class TestEncodingDecoding extends TomcatBaseTest {
             configurator=SingletonConfigurator.class)
     public static class MessagesServer {
 
-        private Queue<String> received = new ConcurrentLinkedQueue<>();
-        private Throwable t = null; 
+        private final Queue<String> received = new ConcurrentLinkedQueue<>();
+        private volatile Throwable t = null;
 
         @OnMessage
         public String onMessage(String message, Session session) {
@@ -431,8 +432,8 @@ public class TestEncodingDecoding extends TomcatBaseTest {
             configurator=SingletonConfigurator.class)
     public static class BatchedServer {
 
-        private Queue<String> received = new ConcurrentLinkedQueue<>();
-        private Throwable t = null; 
+        private final Queue<String> received = new ConcurrentLinkedQueue<>();
+        private volatile Throwable t = null;
 
         @OnMessage
         public String onMessage(String message, Session session) throws IOException {
